Dockerfile

```diff
@@ -1,31 +1,27 @@
-FROM ubuntu:16.04
+# To run: docker run -v /path/to/wsgi.py:/var/www/indexd/wsgi.py --name=indexd -p 81:80 indexd
+# To check running container: docker exec -it indexd /bin/bash 
+
+FROM quay.io/cdis/py27base:pybase2-1.0.1
+
 MAINTAINER CDIS <cdissupport@opensciencedatacloud.org>
 
-RUN apt-get update && apt-get install -y sudo python-pip git python-dev libpq-dev apache2 libapache2-mod-wsgi curl vim libssl-dev libffi-dev \ 
- && apt-get clean && apt-get autoremove \
- && rm -rf /var/lib/apt/lists/*
+RUN mkdir /var/www/indexd \
+	&& chown www-data /var/www/indexd
+
 COPY . /indexd
+COPY ./deployment/uwsgi/uwsgi.ini /etc/uwsgi/uwsgi.ini
+
 WORKDIR /indexd
+
+RUN python -m pip install -r requirements.txt
 RUN COMMIT=`git rev-parse HEAD` && echo "COMMIT=\"${COMMIT}\"" >indexd/index/version_data.py
 RUN VERSION=`git describe --always --tags` && echo "VERSION=\"${VERSION}\"" >>indexd/index/version_data.py
 RUN python setup.py install
 
-RUN mkdir -p /var/www/indexd/ && chmod 777 /var/www/indexd && cp /indexd/wsgi.py /var/www/indexd/wsgi.py && cp /indexd/bin/indexd /var/www/indexd/indexd
-
-#
-# Custom apache2 logging - see http://www.loadbalancer.org/blog/apache-and-x-forwarded-for-headers/
-#
-COPY ./deployment/apache-indexd.conf /etc/apache2/sites-available/apache-indexd.conf
-
-RUN a2ensite apache-indexd
-RUN a2enmod headers
-RUN a2enmod reqtimeout
-RUN a2dissite 000-default.conf
 
 EXPOSE 80
 
 WORKDIR /var/www/indexd
+RUN cp /indexd/wsgi.py .
```

Comment:
Is it okay to point the `wsgi-file` in `uwsgi.ini` to `/indexd/wsgi.py` directly?

Reply:
Moved the location `uwsgi.ini` points to `wsgi.pi`